src/Pages/Profile/MemberView/InfoCard.js

```diff
@@ -67,8 +75,16 @@ export default function ProfileCard(props) {
             }
           </h3>
         ))}
+        <Button
+          className='discord-button'
+          onClick={buttonClick}>
+          <div className='center-text'>
+            <img alt='disLogo' src='https://www.freeiconspng.com/uploads/discord-black-icon-1.png'></img>
```

Comment:
```suggestion
            <img alt='discord logo' src='https://www.freeiconspng.com/uploads/discord-black-icon-1.png'></img>
```

Reply:
how could I make it so that the length of the line doesn't trigger lint? putting `alt` and `src` on different lines doesn't help as `src` part still exceeds char limit

Reply:
```jsx
<img
  alt='disLogo'
  src=`${DISCORD_ICON_URL}`
/>
```

Reply:
just had an oopsie here with spacing